frontend/src/lib/api.ts

```diff
@@ -288,6 +277,24 @@ export function useLibraries(): LibraryVersions[] {
     return data.libraries
 }
 
+export function usePresets(platform: string | undefined): Preset[] {
+    const getByPlatform = ([url, platform]) => {
+        return get(url && platform && `${url}?platform=${platform}&page_size=100`)
```

Comment:
I just realized the site will cease to show any more than 100 presets with this change. I think what we eventually want is a searchable thing with auto-populated results, and then an "Add new..." for creating a new preset 

For now, instead of usePresets, we might want to keep what we had before, so we're sure not to omit any presets from the dropdown

Reply:
It's 100 presets *per platform*, which I think will be fine until we get around to revamping it (i.e. when we let users create presets)

Reply:
Assert that the length of presets is < 100? Currently we have 43 presets for N64 (which I'm assuming is the most popular platform). I'll create an issue to make the Presets a searchable drop-down - I feel it's likely we'll resolve that before we hit 100 presets for any individual platform.

Reply:
Created https://github.com/decompme/decomp.me/issues/1152.